components/cookie-consent/cookie-consent.tsx

```
@@ -0,0 +1,34 @@
+      <Button
+        type="button"
+        variant={"link"}
+        onClick={cookieConsent.showPreferences}
+      >
+        Manage cookie preferences
+      </Button>
+      <Button type="button" variant={"link"} onClick={resetCookieConsent}>
+        Reset cookie consent
+      </Button>
+    </div>
+  ) : null;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)